server/utils/coreErrors.js

```diff
@@ -47,6 +47,13 @@ class AbortScene extends Error {
   }
 }
 
+class BadOauth2ClientResponse extends Error {
```

Comment:
Can this be put in a constants file in the withings integration?

Reply:
I put it in withings integration